ConsoleApp1/Program.cs

```diff
@@ -1,7 +1,32 @@
-﻿namespace ConsoleApp1 {
-    internal class Program {
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
-    }
-}
\ No newline at end of file
+﻿using System;
+
+namespace PalindromeChecker {
+   class Program {
+      static bool IsPalindrome (string input) {
+         input = input.ToLower ().Replace (" ", "").Replace (".", "").Replace (",", "").Replace ("?", "").Replace ("'", "").Replace ("!", "");
+         int left = 0;
+         int right = input.Length - 1;
+
+         while (left < right) {
+            if (input[left] != input[right]) {

```

Comment:
The enclosing brackets can be removed if there is only one statement inside them
for eg.
if (input[left] != input[right] 
   return false;

Reply:
Will update the changes.